Shared/IISFunctions/Get-ApplicationHostConfig.ps1

```diff
@@ -0,0 +1,18 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-ApplicationHostConfig {
+    [CmdletBinding()]
+    [OutputType([System.Xml.XmlNode])]
+    param ()
+
```

Comment:
Need to pass `$CatchActionFunction` so this can be called from Health Checker

Reply:
`Get-ApplicationHostConfig` is called via `Invoke-ScriptBlockHandler` to keep the logic the same as what we have in the other project. We can change it later, what do you think?

<img width="640" alt="image" src="https://user-images.githubusercontent.com/40993616/182907349-43a491ae-49de-4bbd-82b3-743a80e0d51a.png">